openreview/venue_request/process/deployProcess.py

```diff
@@ -320,3 +323,63 @@ def process(client, note, invitation):
         },
         signatures=['~Super_User1']
     ))
+
+    if (forum.content.get('Submission Deadline', None) and forum.content.get('Paper Matching', None)):
+        activation_date = forum.content.get('Submission Deadline').strip()
+        try:
+            activation_date = datetime.strptime(activation_date, '%Y/%m/%d %H:%M')
+        except ValueError:
+            activation_date = datetime.strptime(activation_date, '%Y/%m/%d')
+        matching_invitation = openreview.Invitation(
+            id = SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Matching_Stage',
+            super = SUPPORT_GROUP + '/-/Matching_Stage',
+            invitees = readers,
+            cdate = tools.datetime_millis(activation_date),
+            reply = {
+                'forum': forum.id,
+                'replyto': forum.id,
+                'readers' : {
+                    'description': 'The users who will be allowed to read the above content.',
+                    'values' : readers
+                },
+                'writers': {
+                    'values':[],
+                },
+                'content': {
+                    'title': {
+                        'value': 'Matcher',
+                        'required': True,
+                        'order': 1
+                    },
+                    'matching_group': {
+                        'description': 'Please select the group you want to set up matching for.',
+                        'value-radio': ['reviewer'],
```

Comment:
Can you define the roles directly here?

```
'value-radio': ['reviewer', 'area chair'] if forum.content.get('Area Chairs (Metareviewers)') == "Yes, our venue has Area Chairs" else ['reviewer']
```

What about Senior Area Chairs? I think the setup matching works for that? I'm not sure if conflicts and scores are supported.

Also I would like to start using the name of the group so we don't need to do any translation:

VENUE_ID/Reviewers
VENUE_ID/Area_Chairs

if you use value-dropdown the options are formatted with prettyId so it would say 'NeurIPS 2021 Conference Reviewers'.

If you think that is too large the option then we can remove the venue id and add it later.